eslint.config.js

```diff
@@ -27,7 +27,7 @@ export default [
 			}
 		},
 		rules: {
-			'no-console': 'warn',
+			'no-console': ['warn', { allow: ['warn', 'error'] }],
```

Comment:
Lets not allow these...

Reply:
What should we do then? Don't we want to have some logging for troubleshooting?